src/main/java/pl/mantiscrab/budgetr/registration/User.java

```diff
@@ -15,17 +16,27 @@
 @Table(name = "users")
 class User {
     @Id
-    @GeneratedValue(strategy = GenerationType.IDENTITY)
-    private Long id;
+    @Column(nullable = false)
+    @NotBlank
+    private String username;
+
     @Column(nullable = false, unique = true)
     @Email
     @NotBlank
     private String email;
-    @Column(nullable = false, unique = true)
-    @NotBlank
-    private String username;
+
     @Column(nullable = false)
     @NotBlank
     private String password;
 
+    @Column(nullable = false)
+    private boolean enabled;
+
+    @OneToMany(mappedBy = "user", cascade = CascadeType.ALL)
+    @ToString.Exclude
+    private Set<Authority> authorities;
```

Comment:
don't really get the idea about the authorities

Reply:
:+1: I meant about way of implementing authorities :-) But I think it's quite good option so far :-)